.github/workflows/build.yml

```diff
@@ -24,7 +24,7 @@ jobs:
             fail-fast: false
             matrix:
                 php: ["8.0"]
-                symfony: ["^4.4", "^5.4"]
+                symfony: ["^5.4"]
```

Comment:
This change should be done after branching v1.11. Please revert

Reply:
@lchrusciel Hmmmmmm, so you're suggesting to keep same requirements in Standard and Sylius?

I thought Standard is meant to be a fork-and-forget template for new projects, and thus there was no reason to keep same "relaxed" requirements in it as in sylius/sylius (which can be used during upgrades from older versions, and so it makes sense to support the older LTS there). Is there a flaw in that way of thinking?

Reply:
That makes sense. Indeed, on new project, starting from 4.4 is a no way to go. @lchrusciel WDTY?

Reply:
Are you sure, that everyone may start with Sf 5.4 deps and the related rest? For all these years related Sylius-Standard template had the same constraints as the main Sylius package. I'm not insisting on this tho

Reply:
@vvasiloi Do you have an opinion for that?

Reply:
I agree with Lukasz. Testing Sylius-Standard with Symfony 4 will catch issues the users can have when updating an older project where Sylius was updated, but Symfony wasn't.
Sometimes it's difficult to update both at once because of other dependencies, both own and 3rd party.

Reply:
That is a fair point. I'll update the PR.